test/logger.js

```diff
@@ -31,6 +31,9 @@ describe('Test all Levels', function() {
     let allLevelsServer;
     let sentLines = [];
     let sentLevels = [];
+
+    let callbackResult;
+    const testCallback = (er, res) => { callbackResult = res;};
```

Comment:
Correct me if I'm wrong, but it appears the only place the callback is actually called (and not just passed along) is in `Logger._flush()`, and that seems to call `callback` without any params, so how does this work?

Reply:
@dchai76 it attaches the callback to the instance line 340 and in the async function send_request it calls  the callback that was passed specifically for that instance.